.travis.yml

```diff
@@ -90,6 +90,7 @@ script:
      --enable-remotebackend-zeromq
      --enable-experimental-ed25519
      --enable-experimental-pkcs11
+     --enable-tools
```

Comment:
all tools are build on line 106 and if you enable the toolsbuild there you must remove line 108 (all marked with #DNSName)